tgui/packages/tgui/interfaces/LongRangeTeleporter.js

```diff
@@ -0,0 +1,49 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+export const LongRangeTeleporter = (props, context) => {
+  const { act, data } = useBackend(context);
+
+  const {
+    destinations,
+    receive_allowed,
+    send_allowed,
+    syndicate,
+  } = data;
+
+  return (
+    <Window
+      theme={syndicate ? 'syndicate' : 'ntos'}
+      width={390}
+      height={380}>
+      <Window.Content>
+        <Section title="Destinations">
+          <LabeledList>
+            {destinations.length ? destinations.map((d) => (
+              <LabeledList.Item label={d["destination"]} key={d["destination"]+d["ref"]}>
+                {send_allowed && (
+                  <Button
+                    icon="sign-out-alt"
+                    onClick={() => act("send", { target: d["ref"], name: d["destination"] })} >
+                    Send
+                  </Button>
+                )}
+                {receive_allowed && (
+                  <Button
+                    icon="sign-in-alt"
+                    onClick={() => act("receive", { target: d["ref"], name: d["destination"] })} >
```

Comment:
```suggestion
                    onClick={() => act("receive", { target: d["ref"], name: d["destination"] })}
                  >
```
Pedantic code styling. Could also have it same-line, but the space before the `>` is *weird*.